perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -4452,4 +4452,14 @@ private static boolean checkAuthValidityForMFA(PerunSession sess) {
 	private static boolean sessionHasMfa(PerunSession sess) {
 		return sess.getPerunPrincipal().getAdditionalInformations().containsKey(ACR_MFA);
 	}
+
+	/**
+	 * Returns PerunPolicy object by its name.
+	 *
+	 * @param name - name of the policy
+	 * @return PerunPolicy object
+	 */
+	public static PerunPolicy getPolicyByName(String name) {
```

Comment:
I know that in this class is not the best code organization, but I would move this public method higher in the code between other general methods (it is often mixed in this class, but we usually put private methods at the end of the class).